docs/cli-reference/ks_show.md

```diff
@@ -1,14 +1,36 @@
 ## ks show
 
-Show expanded resource definitions
+Show expanded resource definitions for a specific environment.
 
 ### Synopsis
 
 
-Show expanded resource definitions
+Show expanded resource definitions for a specific environment. Jsonnet manifests
+are expanded into their JSON or YAML equivalents (YAML is the default), and parameters
+from the specified environment are used to fill in any components during this process.
 
+When NO file is specified via the -f flag, this command expands all of the files in the `components/` directory into a list of resource definitions.
```

Comment:
backticks around `-f` (it will also become the `-c` flag, standing for the component name -- so we probably need to change the language around the concept of "files")